modules/api/test/ApiTest.php

```diff
@@ -0,0 +1,388 @@
+<?php declare(strict_types=1);
+/**
+ * This contains tests relevant to the API
+ *
+ * PHP Version 7
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Login
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+
+namespace LORIS\api\Test;
+ini_set('memory_limit', '1024M');
+
+require './vendor/autoload.php';
```

Comment:
This is not a stand alone script so you don't need to do that. It will be called by phpunit which is calling test/bootstrap.php which as that line (the require ... autoloader.php)

What is missing though is :
`require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";`

Because you want this class to extend LorisIntegrationTest